packages/react-dom/src/__tests__/ReactServerRendering-test.js

```diff
@@ -609,6 +609,41 @@ describe('ReactDOMServer', () => {
         expect(response.read()).toBeNull();
       });
     });
+
+    it('should refer users to new apis when using suspense', async () => {
```

Comment:
this test already existed it was just in the wrong describe block. It now correctly lives in the `renderToNodeStream` block